.github/workflows/ci.yml

```
@@ -87,8 +99,8 @@ jobs:
 
       - name: Install dependencies
         run: |
-          pip install $(grep -E '^(invoke)' reqs/requirements-dev-contrib.txt)
+          pip install -c constraints-dev.txt ".[test]"
           invoke deps --gx-install -m '${{ matrix.markers }}' -r test
```

```suggestion
          pip install $(grep -E '^(invoke)' reqs/requirements-dev-contrib.txt)
          invoke deps --gx-install -m '${{ matrix.markers }}' -r test
```
This line defeats the purpose of `invoke deps`, invoke `deps` uses the contraints file be default when installing dependencies.

https://github.com/great-expectations/great_expectations/blob/f51c946b14403352ee4e2573b4b8906859c2fc50/tasks.py#L848-L849

---

@Kilo59 I was seeing this error before the change

```
##[debug]'
##[debug]Loading env
Run pip install $(grep -E '^(invoke)' reqs/requirements-dev-test.txt)
##[debug]/usr/bin/bash -e /home/runner/work/_temp/c52f66bd-5158-456d-930b-fd0f75576e[40](https://github.com/great-expectations/great_expectations/actions/runs/5602528262/jobs/10247904502#step:5:41).sh
ERROR: You must give at least one requirement to install (see "pip help install")
```

---

updating now. There may still be some errors that need to be resolved with the install dependencies stage

---

it worked! Thank you @Kilo59